testUtils/modelMocks.js

```diff
@@ -1,11 +1,16 @@
 import { AuthorizationTypes } from 'shared/constants';
-import * as userSelectors from 'stores/user/selectors';
 
 jest.mock('model/services/giantSwarm/info');
 jest.mock('model/services/metadata/configuration');
-jest.mock('stores/user/selectors');
+jest.mock('stores/main/selectors', () => mockMainSelectors());
 
-// eslint-disable-next-line no-import-assign
-userSelectors.selectAuthToken = jest.fn(() => (state) => {
-  return [state.main.loggedInUser.auth.token, AuthorizationTypes.GS];
-});
+function mockMainSelectors() {
+  const original = jest.requireActual('stores/main/selectors');
+
+  return {
+    ...original,
+    selectAuthToken: jest.fn(() => (state) => {
```

Comment:
Only mocking the `selectAuthToken` function in all tests, so no more random tests failing because of some mocked selector 👍🏻